pmlc/rt/jit_executable.cc

```diff
@@ -388,26 +418,28 @@ class JitExecutable final : public Executable {
     }
 
     setupTargetTriple(llvmModule.get());
-    auto entryPoint = packFunctionArguments(llvmModule.get(), program->entry);
+    packFunctionArguments(llvmModule.get(), util::kPlaidmlExec);
 
     if (VLOG_IS_ON(6)) {
       llvmModule->print(llvm::errs(), nullptr);
     }
 
-    jitEntry = impl->compile(std::move(llvmModule), std::move(ctx), entryPoint);
-    if (!jitEntry) {
-      throw std::runtime_error("jitEntry function is null");
+    abi = impl->compile(std::move(llvmModule), std::move(ctx));
+    if (!abi) {
+      throw std::runtime_error("Entrypoint functions not found");
     }
   }
```

Comment:
Do reduce duplication, let's build up the ABI here, after `impl->compile` so it's part of the common code path.